src/main/java/reactor/netty/channel/MicrometerChannelMetricsRecorder.java

```diff
@@ -76,94 +79,73 @@ protected MicrometerChannelMetricsRecorder(String name, @Nullable String remoteA
 				       .description("Number of the errors that are occurred")
 				       .tag(URI, protocol);
 
-		if (remoteAddress != null) {
-			this.dataReceivedBuilder.tag(REMOTE_ADDRESS, remoteAddress);
-			this.dataReceived = dataReceivedBuilder.register(registry);
+		this.connectTimeBuilder =
+				Timer.builder(name + CONNECT_TIME)
+				     .description("Time that is spent for connecting to the remote address");
 
-			this.dataSentBuilder.tag(REMOTE_ADDRESS, remoteAddress);
-			this.dataSent = dataSentBuilder.register(registry);
+		this.tlsHandshakeTimeBuilder =
+				Timer.builder(name + TLS_HANDSHAKE_TIME)
+				     .description("Time that is spent for TLS handshake");
 
-			this.errorCountBuilder.tag(REMOTE_ADDRESS, remoteAddress);
-			this.errorCount = errorCountBuilder.register(registry);
-		}
+		this.addressResolverTimeBuilder =
+				Timer.builder("reactor.netty.address.resolver")
```

Comment:
Should **_name_** be set for the **_reactor.netty.address.resolver_** metric (_Timer.builder(**_name_** + "address.resolver")_) or is it deliberately not set for some reason?

Reply:
ok let's use the name also for the DNS resolver